src/content/docs/en/install/manual.mdx

```diff
@@ -12,7 +12,9 @@ This guide will walk you through the steps to manually install and configure a n
 
 #### Prefer a quicker way to get started?
 
+<p>
 <Button href="/en/install/auto/">Try the create astro CLI wizard →</Button>
+</p>
```

Comment:
Thanks for spotting this @liruifengv! I wonder if actually the button treatment can be dropped now in favour of something more like what we have on https://docs.astro.build/en/install/auto/ ?

```suggestion
:::tip[Prefer a quicker way to get started?]
Follow the [create astro CLI wizard](/en/install/auto/) instead.
:::
```

Reply:
It's great!